cl/api/tests.py

```diff
@@ -22,6 +22,7 @@
 from django.test.utils import CaptureQueriesContext
 from django.urls import reverse
 from django.utils.timezone import now
+from rest_framework import status
```

Comment:
This used to be needed back in the day, but Python packaged these into the http lib (I think), so it's probably better to use that since it's more official. At some point I think I swapped all of our existing imports to use it, IIRC.